lib/src/main/java/com/uber/cherami/client/metrics/MetricsReporter.java

```diff
@@ -111,6 +111,8 @@ public LatencyProfiler createLatencyProfiler(Latency latencyMetric) {
         PUBLISHER_RECONFIGS("cherami.publish.reconfigure.rate"),
         /** Number of messages received by the consumer. */
         CONSUMER_MESSAGES_IN("cherami.consume.message.rate"),
+        /** Number of consume message attempts that failed or timed out */
```

Comment:
I don't think you know if the message timed out, at least for the consume side. The server maintains the timer, and it doesn't have any feedback.